config/config.json.example

```diff
@@ -1,16 +1,30 @@
-{
-    "moderator_channel": "moderation-logs",
-    "support_channel": "support",
-    "rules_channel": "rules",
-    "non_support_channels": [
-        "general"
-    ],
-    "bad_words": [
-        "bum",
-        "hole"
-    ],
-    "appeal_url": "https://www.google.com",
-    "client_id": "",
-    "client_secret": "",
-    "client_token": ""
-}
+{
+    "moderator_channel": "moderation-logs",
+    "support_channel": "support",
+    "rules_channel": "rules",
+    "non_support_channels": [
+        "general"
+    ],
+    "moderated_channels": [
+        "general",
+        "support"
+    ],
+    "bypass": {
+        "roles": [
+            "ATLauncher Staff",

```

Comment:
Can we change this to simply just include one role with a name of "Staff" just as an example so it's not 'ATLauncher' specific

Reply:
changed to owner and moderator to indicate comma list. jsonlint cleared errors